libs/shared-atomic-design-components/src/lib/molecules/Paginate.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+} from '@talent-connect/shared-shadcn-ui-components'
+import './Paginate.scss'
+
+interface PaginateProps {
+  totalPagesNumber: number
+  currentPageNumber: number
+  setCurrentPageNumber: (page: number) => void
+}
+
+type ActivePagesType = number | 'ellipsis'
+
+const MAX_VISIBLE_PAGES = 5
+
+const Paginate = ({
+  totalPagesNumber,
+  currentPageNumber,
+  setCurrentPageNumber,
+}: PaginateProps) => {
+  if (!totalPagesNumber) return null
+
+  const scrollToListTop = () => {
+    window.scrollTo({ top: 350, behavior: 'smooth' })
+  }
```

Comment:
As coderabbit suggests, does 350 also work for mobile or other display sizes?

Reply:
@helloanil, yes. I played around a bit with it to find the one that looks good on all display sizes.

Reply:
Good, but .... does that only apply to the single page where this component is being used? What about when we want to use the component in other contexts? The component doesn't look super reusable to me at the moment.

Reply:
What exactly do you think is not reusable?

I understood from Ligia's design that there should always be five pagination items.

Reply:
The `350` value used in the `scrollToTop` function call. It implies that the pagination component can only be used on a page where the paginated elements start appearing at a precise vertical offset.

Reply:
Thoughts?

Reply:
I didn't reply here because we are considering removing the `scrollToListTop` function.

But I got your point 👍 If we want to use this component in other places, it's better to pass this value as a prop to it.

Reply:
Had a call about it